packages/form/src/form.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { FC, useState } from 'react';
+import { FormProps, FieldProps } from "./props";
+import TextInput from "@openstad-headless/ui/src/form-elements/text";
+import RangeSlider from "@openstad-headless/ui/src/form-elements/a-b-slider";
+import CheckboxField from "@openstad-headless/ui/src/form-elements/checkbox";
+import RadioboxField from "@openstad-headless/ui/src/form-elements/radio";
+import SelectField from "@openstad-headless/ui/src/form-elements/select";
+import TickmarkSlider from "@openstad-headless/ui/src/form-elements/tickmark-slider";
+import FileUploadField from "@openstad-headless/ui/src/form-elements/file-upload";
+import MapField from "@openstad-headless/ui/src/form-elements/map";
+import { handleSubmit } from "./submit";
+
+const Form: FC<FormProps> = ({
+     title = 'Form Widget',
+     fields = [],
+     submitText = 'Verzenden',
+     submitHandler = () => {},
```

Comment:
In de oude openstad hadden we bij bijvoorbeeld de resource-form de mogelijkheid om een "opslaan als concept" knop toe te voegen als deze voor het aanmaken van een nieuwe resource gebruikt wordt. 
Kan er worden toegevoegd dat er een array van submittext en handlers kunnen worden opgegeven?